x11-terms/mlterm/mlterm-3.9.1.ebuild

```diff
@@ -12,12 +12,11 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="bidi brltty cairo canna debug fbcon fcitx freewnn gtk gtk2 harfbuzz ibus libssh2 m17n-lib nls regis scim skk static-libs uim utempter wayland xft"
+IUSE="X bidi brltty cairo canna debug fbcon fcitx freewnn gtk gtk2 harfbuzz ibus libssh2 m17n-lib nls regis scim skk static-libs uim utempter wayland xft"
 REQUIRED_USE="gtk2? ( gtk )"
```

Comment:
Need `|| ( X fbcon wayland )` and a default (`+X` would make sense to preserve old behavior), if all are unset it tries to use automagic.

Reply:
Done